fix: implement changes keeping metatags (#1078)
aziontech/docs: src/content/docs/en/pages/build-journey/edit-edge-app/edit-device-groups/device-groups.mdx

---
title: How to create device groups

> description: Learn how to create and manage Device Groups in Azion Console to tailor

Reviewer: ```suggestion
description: Learn how to create and manage Device Groups on Azion to tailor
```
